cmd/pilot-agent/main.go

```diff
@@ -221,6 +251,10 @@ func init() {
 		"IP Address and Port of a statsd UDP listener (e.g. 10.75.241.127:9125)")
 	proxyCmd.PersistentFlags().IntVar(&proxyAdminPort, "proxyAdminPort", int(values.ProxyAdminPort),
 		"Port on which Envoy should listen for administrative commands")
+	proxyCmd.PersistentFlags().IntVar(&controlPlaneAuthPolicy, "controlPlaneAuthPolicy",
+		int(values.ControlPlaneAuthPolicy), "Enum value of controlPlaneAuthPolicy")
```

Comment:
okay, now I see the disconnect. For clarity, I suggest changing this to a text based option list, like the service registries.. enable, disable. Or even simply, --enable-control-plane-auth. you dont have to block on updating api..